.travis/install_prlimit.sh

```diff
@@ -15,17 +15,29 @@
 
 set -e
 
-OUT_DIR=$1
-
-pushd $PWD
+BUILD_DIR=$1
+INSTALL_DIR=$2
 
+cd $BUILD_DIR
 wget https://www.kernel.org/pub/linux/utils/util-linux/v2.25/util-linux-2.25.2.tar.gz
 tar -xzvf util-linux-2.25.2.tar.gz
 cd util-linux-2.25.2
-./configure ADJTIME_PATH=/var/lib/hwclock/adjtime --disable-chfn-chsh --disable-login --disable-nologin --disable-su --disable-setpriv --disable-runuser --disable-pylibmount --disable-static --without-python --without-systemd --without-systemdsystemunitdir --without-ncurses || cat config.log
+./configure ADJTIME_PATH=/var/lib/hwclock/adjtime \
+    --disable-chfn-chsh \
+    --disable-login \
+    --disable-nologin \
+    --disable-su \
+    --disable-setpriv \
+    --disable-runuser \
+    --disable-pylibmount \
+    --disable-static \
+    --without-python \
+    --without-systemd \
+    --disable-makeinstall-chown \
+    --without-systemdsystemunitdir \
+    --without-ncurses \
+    --prefix=$INSTALL_DIR || cat config.log
 
-# only compile prlimit
-make prlimit
-mv ./prlimit $OUT_DIR
+make -j `grep -c ^processor /proc/cpuinfo`
```

Comment:
`nproc` is simpler and also works. See: https://github.com/awslabs/s2n/blob/master/tests/fuzz/runFuzzTest.sh#L40